### PR TITLE
LPS-103534 DBInitUtil

### PR DESCRIPTION
**Pull request by Preston-Crary**

### CODE REVIEW

**Comment by Preston-Crary** [comment]
ci:test:backend

**Comment by Preston-Crary** [comment]
ci:test:upgrade

File: portal-impl/src/com/liferay/portal/service/impl/ReleaseLocalServiceImpl.java [hunk 1]
```diff
@@ -188,47 +193,22 @@ public int getBuildNumberOrCreate() throws PortalException {
...
-
-			if (rs.next()) {
-				buildNumber = rs.getInt("buildNumber");
-			}
-			else {
-				buildNumber = _addReleaseInfo();
-			}
-
+		try {
+			db.runSQL("alter table Release_ add state_ INTEGER");
```
Comment (shuyangzhou) [inline portal-impl/src/com/liferay/portal/service/impl/ReleaseLocalServiceImpl.java]: @Preston-Crary the state_ adding was done in 2/28/2012, the schemaVersion adding was done in 8/22/2015. Both are very old, it should be safe to assume all customers having them already. Another way to look at it, we don't support that super old version directly upgrade to 7.2/7.3. If they do the upgrade by steps, they will have those columns added anyway. Let's just remove these lazy adding logic to keep things simple. Please resend, thanks!

**Comment by shuyangzhou** [comment]
See my comment above, please resend, thanks!

**Comment by Preston-Crary** [comment]
Please see https://github.com/shuyangzhou/liferay-portal/pull/8444

**Pull request closed without merging**